src/features/bookings/helpers/expirationDateUtils.test.ts

```diff
@@ -98,4 +99,19 @@ describe('expirationDateUtils', () => {
       expect(daysCountdown(dateCreated)).toEqual(endedCountdown)
     })
   })
+
+  describe('formattedExpirationDate', () => {
+    it('should formatted expiration date from the created date', () => {
+      const dateCreated = '2023-01-19T14:38:45Z'
+      const expectedformattedExpirationDate = '18/02/2023'
+
+      expect(formattedExpirationDate(dateCreated)).toEqual(expectedformattedExpirationDate)
+    })
+
+    it('should return undefined if dateCreated is an empty string', () => {
```

Comment:
C'est plus `should return an empty string when dateCreated is an empty string` mais c'est du détail